src/metabase/driver/presto.clj

```diff
@@ -124,7 +125,7 @@
     (if (= v "NaN") 0.0 v)))
 
 (defn- analyze-table [driver table new-table-ids]
-  ((analyze/make-analyze-table driver
+  ((cached-values/make-field-extractor driver
      :field-avg-length-fn   field-avg-length
      :field-percent-urls-fn field-percent-urls) driver table new-table-ids))
```

Comment:
Wait, `make-field-extractor` doesn't take `field-avg-length-fn` and `field-percent-urls-fn` anymore, right? This looks wrong